Sources/ClientRuntime/Config/DefaultClientConfiguration.swift

```diff
@@ -5,6 +5,8 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+import struct SmithyRetriesAPI.RetryStrategyOptions
+
```

Comment:
Separate imports have to be added in many files for the retry types extracted into modules.
I won't comment in these files every time those are added.